addons/gradein/models/gradein_order.py

```diff
@@ -54,7 +55,14 @@ class GradeInOrder(models.Model):
         string="Respuestas",
         required=True,
     )
-
+    
+    @api.constrains("question_answer_id")
+    def validate_answers(self):
+        
+        questions_answers = self.question_answer_id
+        for record in questions_answers:
+            if record.answer_id.blocking == True:
```

Comment:
Cuando se compara un booleano en Python, lo correcto es usar el keyword `is`, en este caso se puede simplificar de la siguiente manera:
```python
if record.answer_id.blocking:
   # code
```

Reply:
modificado :+1: